### PR TITLE
Add continuous integration

### PR DESCRIPTION
**Pull request by saschagrunert**
I just avoided adding the makefile and the stuff but for now it works as intended.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    <<: *defaults
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - 90:8c:cc:9d:df:00:3c:82:33:a7:13:5e:64:9b:ed:73
+      - checkout
+      - run:
+          name: Setup git
+          command: |
+            git config --global user.email mail@saschagrunert.de
```
Comment (niko2342) [inline .circleci/config.yml]: can we use another mail address for this repo?

Comment (saschagrunert) [reply .circleci/config.yml]: Changed it

**Comment by saschagrunert** [comment]
@niko2342 Please review again. Another option for later usage would be to use [bors](https://github.com/graydon/bors)

**Comment by niko2342** [comment]
Closes #12

**Pull request merged**